.dev-lib

```diff
@@ -0,0 +1,8 @@
+DEFAULT_BASE_BRANCH=develop
+PHPCS_GIT_TREE=master
+PHPCS_IGNORE='tests/*,dev-lib/*,includes/vendor/*'
+PHPCS_RULESET_FILE=./.phpcs.xml.dist
```

Comment:
Why not just rename `.phpcs.xml.dist` to `phpcs.xml.dist` so that PHPCS will pick it up by default?

Reply:
Because PHPCS (as of v 3.1.0) will pick up `.phpcs,xml.dist` files by default as well.... With the `.` prefix, the file now groups itself with other similar configuration files, so when given the choice, using the `.` prefix is recommended.

As a side-note: the order of preference by PHPCS (as of 3.1.1) is `.phpcs.xml`, `phpcs.xml`, `.phpcs.xml.dist`, `phpcs.xml.dist`.
So using the `.` also means the file will be found sooner.

Reply:
Yeah, I missed your other PR: https://github.com/xwp/wp-dev-lib/pull/266

Reply:
No problem. Glad that's cleared up ;-)